v3/src/components/data-display/components/legend/categorical-legend.tsx

```diff
@@ -349,7 +349,7 @@ export const CategoricalLegend = observer(
     }, [setDesiredExtent, layerIndex])
 
     return (
-      <svg className='legend-categories' ref={keysElt} data-testid='legend-categories'></svg>
+      <g className='legend-categories' ref={keysElt} data-testid='legend-categories'></g>
```

Comment:
It seemed to me these `legend-categories`items don't need to be `svg` elements themselves since they always seem to be contained within a parent `svg`. I changed them to `g` elements because the `svg`s were causing duplications in the image file exports. If I'm overlooking a reason why these should remain `svg`s, the `graphSnaphsot` function in image-utils.ts will need to be adjusted.

Reply:
Seems okay to me, but @bfinzer should take a look as well.

Reply:
Seems OK to me as well.